### ISSUE TITLE
`actionType` is private

### ISSUE THREAD

**Issue by Primetalk** [slack-go/slack#607]
https://github.com/nlopes/slack/blob/master/attachments.go#L20
This makes it a little tricky to construct an instance of `AttachmentAction`. The only way is to use string literals:
```go
action.Type = "button"
```

**Comment by simonfrey** [comment]
Took me hours to find this issue. Having actionType private is not intuitive. If you want to keep it privat provide at least two public constants with the values
```
const(
  BUTTON actionType = "button"
  SELECT actionType = "select"
)
```

**Comment by GLOFonseca** [comment]
I don't see why this type is private, so I've created a PR to change it.
#831

**Comment by GLOFonseca** [comment]
I'm not a maintainer, so we need someone to review this.

**Comment by GLOFonseca** [comment]
@kanata2 I think you are the only maintainer I know, could you look at this issue, or maybe pin any other maintainer?

**Comment by kanata2** [comment]
Fixed in v0.9.0!
